### PR TITLE
Add optional flags to translate

### PR DESCRIPTION
**Pull request by Soheab**
This adds optional --to and --from flags to the translate command. Used the discord-flags package for this, which i also added to requirements.txt. Also added flags.ArgumentParsingError to the error handler to be handled.

### CODE REVIEW

File: cogs/events.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 import config
 import discord
-from discord.ext import commands, tasks
+from discord.ext import commands, tasks, flags
```
Comment (GavynStanley) [inline cogs/events.py:9]: Why do you need flags in the events file?

File: cogs/staff_site.py [hunk 2]
```diff
@@ -195,27 +195,36 @@ async def say(self, ctx, *, msg: commands.clean_content):
...
             except ValueError:
                 return await ctx.send("That is not a valid language")
-            except Exception as e:
+            except Exception:
                 translator = googletrans.Translator()
         src = googletrans.LANGUAGES.get(
             translated.src, '(auto-detected)').title()
         dest = googletrans.LANGUAGES.get(translated.dest, 'Unknown').title()
         embed = discord.Embed(color=discord.Color.blurple())
-        embed.add_field(name=f"{src} ({translated.src})",
```
Comment (GavynStanley) [inline cogs/staff_site.py:215]: Stop removing the formatting. I am doing this to please VSC

**Review by GavynStanley — CHANGES_REQUESTED** [review]

File: cogs/events.py (hunk 1 above)
Comment (Soheab) [reply cogs/events.py:9]: see line 60

File: cogs/staff_site.py (hunk 2 above)
Comment (Soheab) [reply cogs/staff_site.py:215]: oooh

File: cogs/events.py (hunk 1 above)
Comment (GavynStanley) [reply cogs/events.py:9]: 👌

**Review by GavynStanley — APPROVED** [review]

**Pull request merged**